v2client/src/main/java/com/yubico/client/v2/YubicoClient.java

```diff
@@ -50,12 +50,9 @@
     protected Integer clientId;
     protected byte[] key;
     protected Integer sync;
+    protected Integer maxRetries = 5;
```

Comment:
This can be an `int`, right?

```suggestion
    protected int maxRetries = 5;
```